### PR TITLE
[KOGITO-1214] - Do not use echo statement on the images to print log …

### PR DESCRIPTION
**Pull request by vaibhavjainwiz**
…messages

### CODE REVIEW

File: modules/kogito-infinispan-properties/tests/bats/kogito-infinispan-properties.bats [hunk 1]
```diff
@@ -1,20 +1,18 @@
...
+export KOGITO_HOME=/tmp/kogito
+mkdir -p ${KOGITO_HOME}/launch
+cp $BATS_TEST_DIRNAME/../../../kogito-logging/added/logging.sh ${KOGITO_HOME}/launch/
 
 # import
 load $BATS_TEST_DIRNAME/../../added/kogito-infinispan-properties.sh
 
-function clear_vars() {
-    unset INFINISPAN_USEAUTH
-    unset INFINISPAN_USERNAME
```
Comment (spolti) [inline modules/kogito-infinispan-properties/tests/bats/kogito-infinispan-properties.bats:10]: why are you removing it?

File: modules/kogito-data-index/tests/bats/kogito-data-index.bats [hunk 2]
```diff
@@ -1,118 +1,23 @@
...
-
 # imports
-load ${KOGITO_HOME}/launch/kogito-infinispan-properties.sh
 load $BATS_TEST_DIRNAME/../../added/launch/kogito-data-index.sh
 
-teardown() {
-    rm -rf ${KOGITO_HOME}
-}
-
-function clear_vars() {
```
Comment (spolti) [inline modules/kogito-data-index/tests/bats/kogito-data-index.bats:16]: why did you remove these tests?

File: modules/kogito-jobs-service/tests/bats/kogito-jobs-service.bats [hunk 3]
```diff
@@ -53,19 +48,13 @@ teardown() {
     [ "${KOGITO_JOBS_PROPS}" = "${expected}" ]
 }
 
-
 @test "check if the persistence is correctly configured with auth" {
     export ENABLE_PERSISTENCE="true"
     export INFINISPAN_CLIENT_SERVER_LIST="localhost:11222"
-    export INFINISPAN_USEAUTH="true"
```
Comment (spolti) [inline modules/kogito-jobs-service/tests/bats/kogito-jobs-service.bats:60]: why removing it?

**Review by spolti — CHANGES_REQUESTED** [review]

File: modules/kogito-infinispan-properties/tests/bats/kogito-infinispan-properties.bats (hunk 1 above)
Comment (vaibhavjainwiz) [reply modules/kogito-infinispan-properties/tests/bats/kogito-infinispan-properties.bats:10]: I replaced it with setup() function which is provided by BATS. Setup function automatically calls before executing every test.

function setup() {
    prepareEnv
    unset INFINISPAN_PROPERTIES
}

File: modules/kogito-data-index/tests/bats/kogito-data-index.bats (hunk 2 above)
Comment (vaibhavjainwiz) [reply modules/kogito-data-index/tests/bats/kogito-data-index.bats:16]: I had removed some bats test from kogito-data-index.bats file because these are redundant. kogito-data-index file contains some tests of kogito-infinispan-properties which are already covered in kogito-infinispan-properties.bats file

File: modules/kogito-jobs-service/tests/bats/kogito-jobs-service.bats (hunk 3 above)
Comment (vaibhavjainwiz) [reply modules/kogito-jobs-service/tests/bats/kogito-jobs-service.bats:60]: I removed it because it tests kogito-inifinispan-properties which are already tested in kogito-infinispan-properties.bats file

**Comment by vaibhavjainwiz** [comment]
incorporate review comments and raised new pull request
https://github.com/kiegroup/kogito-images/pull/153
